web/public/gameData_v1.0-18.json

```diff
@@ -189,6 +195,18 @@
                 "isFluid": false,
                 "isFicsmas": false
             },
+            "Crystal_mk2": {
+                "name": "Yellow Power Slug",
+                "stackSize": 50,
+                "isFluid": false,
+                "isFicsmas": false
+            },
+            "Crystal_mk3": {
+                "name": "Purple Power Slug",
+                "stackSize": 50,
+                "isFluid": false,
+                "isFicsmas": false
+            },
```

Comment:
Not sure we should be adding Purple and Green power slugs. The `Crystal` one makes sense as it's used for Synthetic Power Shards, but the others, not sure.

I won't block the PR over it, just a note for the future.